clcache.py

```diff
@@ -1293,6 +1293,12 @@ def analyze(cmdline):
 
 def invokeRealCompiler(compilerBinary, cmdLine, captureOutput=False, outputAsString=True, environment=None):
     realCmdline = [compilerBinary] + cmdLine
+
+    sourceFile = realCmdline[-1]
+    if '/Tc' + sourceFile in realCmdline or '/Tp' + sourceFile in realCmdline:
+        printTraceStatement("Removing last argument becuase of /Tc (Issue #289)")
```

Comment:
Small typo here (`becuase`) and I guess `/Tc` is imprecise because it seems this could also be because of `/Tp`.

Reply:
I'm going to ignore this for now because it's not relevant to the discussion.